.github/workflows/build-aws-hub.yaml

```diff
@@ -0,0 +1,75 @@
+name: Deploy AWS Hub
+on:
+  push:
+    branches:
+      - staging
+      #- prod
+    paths:
+      - 'deployments/icesat2/**'
+      - 'pangeo-deploy/**'
+    paths_ignore:
+      - 'deployments/icesat2/image/binder/*'
+
+env:
+  HUBPLOY_IMAGE: docker://yuvipanda/hubploy:20200826083951674280
+
+jobs:
+  deploy_aws:
+    name: Deploy AWS Hub
+    # This job runs on Linux
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: $HUBPLOY_IMAGE
+        name: Unlock git-crypt Secrets
+        env:
+          GIT_CRYPT_KEY: ${{ secrets.GIT_CRYPT_KEY }}
+        with:
+          entrypoint: /bin/bash
+          args: -c "echo ${GIT_CRYPT_KEY} | base64 -d | git crypt unlock - && git crypt status"
+      #- uses: $HUBPLOY_IMAGE
+      #  name: Build & Push AWS Image if Needed
+      #  with:
+      #    args: -c "build icesat2 --check-registry --push"
+      - uses: $HUBPLOY_IMAGE
+        name: Setup Helm
+        with:
+          entrypoint: /bin/bash
+          args: >
+            -c "helm init --client-only &&
+            helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/ &&
+            helm repo add dask https://helm.dask.org/ &&
+            helm repo add dask-gateway https://dask.org/dask-gateway-helm-repo/ &&
+            helm repo add stable https://kubernetes-charts.storage.googleapis.com &&
+            helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx &&
+            helm repo add prometheus-operator https://kubernetes-charts.storage.googleapis.com &&
+            helm repo update"
+      - uses: $HUBPLOY_IMAGE
+        name: Add Runner IP to EKS Kubernetes API Whitelist
+        with:
+          entrypoint: /bin/bash
+          args: >
+            -c "export AWS_SHARED_CREDENTIALS_FILE=./deployments/icesat2/secrets/aws-config.txt &&
+            RUNNERIP=`curl --silent https://checkip.amazonaws.com` &&
+            aws --version &&
+            aws eks update-cluster-config --region us-west-2 --name pangeo --resources-vpc-config publicAccessCidrs=${RUNNERIP}/32 > /dev/null &&
+            sleep 120"
+      - uses: $HUBPLOY_IMAGE
+        name: Deploy AWS Staging Hub
+        if: github.ref == 'refs/heads/staging'
+        with:
+          args: deploy icesat2 pangeo-deploy staging --timeout 1200s --cleanup-on-fail
+      - uses: $HUBPLOY_IMAGE
+        name: Revert to Original EKS IP Whitelist
```

Comment:
I recommend adding `if: always()` for this step so that we always get back to original list even if previous steps fail

Reply:
Added